Assets/Scripts/PlayerInput.cs

```diff
@@ -81,6 +81,9 @@ void Update()
             // If not currently in astral form, go into astral form
             if (!projecting)
             {
+            	// Nullify player's movement
+            	player.SetDirectionalInput(new Vector2(0f,0f));
```

Comment:
I noticed that when an air rune is rotated to the player's position, the player will move even while projecting. Do we want this behavior or should they be immune to runes while in projection mode? I would say the former, but this might require more discussion

Reply:
I think the player should be immune to the runes while placing them--that way, they don't, for example, missplace an air rune and blow their body away while in astral projection mode.